client/reader/following/main.jsx

```diff
@@ -72,14 +72,14 @@ const FollowingStream = ( props ) => {
 			{ showRegistrationMsg && (
 				<Banner
 					className="following__reader-vote"
-					title="Election Day: Tuesday November 6th"
+					title="NZ Election Day: Saturday 17 October"
 					callToAction="How to vote"
-					description="Remember to vote."
-					dismissPreferenceName="reader-vote-prompt"
```

Comment:
Removed the dismiss button because it's not accessible - will raise an issue for that.

Reply:
https://github.com/Automattic/wp-calypso/issues/46378